networkx/algorithms/operators/tests/test_all.py

```diff
@@ -1,5 +1,6 @@
 import pytest
 import networkx as nx
+from networkx.algorithms.operators.all import relational_compose_all # Why can't this be accessed from nx.relational_compose_all?
```

Comment:
Is it not exposed in the `__init__.py`? Is it missing from some `__all__` registry?

Reply:
Ah, was left out of the __all__ registry.  Thanks.